src/docfx/restore/RestoreGit.cs

```diff
@@ -183,32 +183,26 @@ void AddWorkTrees()
 
         private static IEnumerable<(string remote, string branch, GitFlags flags)> GetGitDependencies(Config config, string locale, Repository rootRepository)
         {
-            var dependencies = config.Dependencies.Values.Select(url =>
+            foreach (var (_, url) in config.Dependencies)
```

Comment:
will this change also applies to Dependencies? How we know the branch info for local dependency folder? Can we add a spec test for that?